scripts/validate_docstrings.py

```diff
@@ -183,20 +182,25 @@ def validate_pep8(self):
             )
         )
 
-        application = flake8.main.application.Application()
-        application.initialize(["--quiet"])
-
+        error_messages = []
         with tempfile.NamedTemporaryFile(mode="w", encoding="utf-8") as file:
             file.write(content)
             file.flush()
-            application.run_checks([file.name])
-
-        # We need this to avoid flake8 printing the names of the files to
-        # the standard output
-        application.formatter.write = lambda line, source: None
-        application.report()
-
-        yield from application.guide.stats.statistics_for("")
+            # GH40784
+            cmd = ["flake8", "--quiet", "--statistics", file.name]
```

Comment:
Can this be
```suggestion
            cmd = ["python", "-m", "flake8", "--quiet", "--statistics", file.name]
```
, to make sure the correct `flake8` is called?